gremlin-go/driver/anonymousTraversal.go

```diff
@@ -22,7 +22,7 @@ package gremlingo
 // AnonymousTraversalSource interface for generating anonymous traversals.
 type AnonymousTraversalSource interface {
 	// WithRemote used to set the DriverRemoteConnection within the AnonymousTraversalSource
-	WithRemote(drc *DriverRemoteConnection) *GraphTraversalSource
+	With(drc *DriverRemoteConnection) *GraphTraversalSource
```

Comment:
Are we just deprecating the usage? If so, we should keep WithRemote() in addition to adding With(), as removing it will remove the method implementation from the interface. I.e.:
```suggestion
	With(drc *DriverRemoteConnection) *GraphTraversalSource
	// WithRemote used to set the DriverRemoteConnection within the AnonymousTraversalSource
	//
	// Deprecated: WithRemote replaced by With
	WithRemote(remote *DriverRemoteConnection) *GraphTraversalSource
```

Reply:
that was a mistake. good catch